docs/configuration.md

```diff
@@ -18,3 +18,7 @@ The following environment variables are available:
 | `BUILD_OPERATOR_LEASE_DURATION` |  Override the `LeaseDuration`, which is the duration that non-leader candidates will wait to force acquire leadership. |
 | `BUILD_OPERATOR_RENEW_DEADLINE` |  Override the `RenewDeadline`, which is the duration that the acting master will retry refreshing leadership before giving up. |
 | `BUILD_OPERATOR_RETRY_PERIOD` |  Override the `RetryPeriod`, which is the duration the LeaderElector clients should wait between tries of actions. |
+| `CONTROLLER_BUILD_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the build controller. Default is 1. |
+| `CONTROLLER_BUILDRUN_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the buildrun controller. Default is 1. |
+| `CONTROLLER_BUILDSTRATEGY_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the buildstrategy controller. Default is 1. |
+| `CONTROLLER_CLUSTERBUILDSTRATEGY_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the clusterbuildstrategy controller. Default is 1. |
```

Comment:
I am thinking if the name of these env properties are too long? Can we remove the`CONTROLLER_` prefix?

Reply:
Based on `BUILD_CONTROLLER_LEASE_DURATION`, I should even further extend it to `BUILD_CONTROLLER_BUILD_MAX_CONCURRENT_RECONCILES` :-) . Anyway, I shortened it. Imo the other env vars could get their `BUILD_CONTROLLER_` prefix removed.

Reply:
Agree, I think we should improve the env proprieties as a same style, and improve this list to show the different usage better. But I think we can do it in another PR.